app/feature.go

```diff
@@ -0,0 +1,133 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag   = "enable-export"
+	featureflagFileName = "feature.json"
+)
+
+type FeatureFlag struct {
+	Name  string `json:"Name"`
+	Value bool   `json:"Value"`
+}
+
+func getFeatureFlagJSON() ([]FeatureFlag, error) {
```

Comment:
[Nit] function is called getFeatureFlagJSON(), but this is just returning a slice of feature flags, so maybe we can just call it getFeatureFlags() ?